ceph.spec.in

```diff
@@ -12,6 +12,10 @@
 %bcond_with selinux
 %endif
 
+# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12
+%if 0%{?fedora} || 0%{?rhel} >= 6 || 0%{?suse_version} == 1315
```

Comment:
i see. that comment is kind of misleading. maybe we can update it in master later on along with some non trivial change to ceph.spec.in?